plugins/outputs/redistimeseries/redists.go

```diff
@@ -0,0 +1,82 @@
+package redistimeseries
+
+import (
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  addr = "127.0.0.1:6379"
+
+  ## password to login Redis
+  # password = ""
+`
+
+type RedisTimeSeries struct {
+	addr     string
+	password string
+	client   *redis.Client
+}
+
+func (i *RedisTimeSeries) Connect() error {
+
```

Comment:
Please skip the superfluous newlines where it doesn't help structuring the code.